### DISCUSSION TITLE
How could I trigger an event after slide change ends?

### DISCUSSION THREAD

**Discussion by ti-mercado-industrial** [ismail9k/vue3-carousel#250] [Q&A] [answered]
Hi, how could I call a method when after slide updates (when the change from one slide to another ends)?

**Comment by ismail9k** [comment, marked as answer]
Please check events page in the docs (https://ismail9k.github.io/vue3-carousel/api/events.html#slide-start)

**Comment by ti-mercado-industrial** [comment]
Thanks! I will check it!

El lun, 28 nov 2022 a la(s) 12:08, Abdelrahman Ismail (
***@***.***) escribió:

> Please check events page in the docs (
> https://ismail9k.github.io/vue3-carousel/api/events.html#slide-start)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ismail9k/vue3-carousel/discussions/250#discussioncomment-4256458>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AT4BPSSLVUZDIBWEPWRSP3DWKT7KPANCNFSM6AAAAAASBHVQPQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
[image: Mercado Industrial] <https://www.mercadoindustrial.com.mx/>

*Ing. Rafael Montoya Boneo*
*Tecnologías de la Información*
*Mercado Industrial *

Calle Héctor Acedo Valenzuela #2
Col. Sector Industrial Presa. C.P. 83293
Hermosillo, Sonora.
Teléfonos. (662)4-60-02-12
www.mercadoindustrial.com.mx


þ *Gracias por imprimir solo lo necesario  -  Please consider the
environment before printing this email*
